test_basic.rb

```diff
@@ -86,6 +86,27 @@ def test_registration
     assert_equal(expected_text, actual_text)
   end
 
+  def test_random_action
+    register_user
+    project_name = 'test_project' + rand(999999).to_s
+    project_description = 'test_description' + rand(9999).to_s
```

Comment:
why we don't move strings' generations to the create_project method?
we won't see them here then at all and this method won't need any params also